tasks/deconst_assets.js

```diff
@@ -80,20 +80,28 @@ module.exports = function(grunt) {
                 headers: {
                     'Authorization': 'deconst ' + ConfigService.get('key')
                 },
-                formData: formData
-            }, function (error, response, body) {
+                formData: formData,
+                timeout: 60000
+            }, function (err, response, body) {
+                if (err) {
+                    grunt.fail.fatal(err);
+                }
+
                 // API key is provided but invalid
                 if (response && response.statusCode === 401) {
                     grunt.fail.fatal("API Key Invalid");
                 }
+                else if (response && response.statusCode !== 200) {
```

Comment:
I thought about that too. I looked at the [source](https://github.com/deconst/content-service/blob/master/src/routes/assets/store.js#L45) and I don't believe it can ever send anything but a `200` in the success case.